client/account.go

```diff
@@ -60,12 +60,21 @@ func (c *Client) GetAccount(userGuid, accountGuid string) (*models.Account, erro
 	return nil, makeGenericError(response.StatusCode, bufferStr)
 }
 
-func (c *Client) ListAccountTransations(userGuid, accountGuid string) ([]*models.Transaction, error) {
+func (c *Client) ListAccountTransactions(userGuid, accountGuid, fromDate, toDate string) ([]*models.Transaction, error) {
```

Comment:
I don't think we should change this interface so it might be good to add an extra method that calls into this new expanded method. Since we can't overload method names, we can rename the date range method like this:

```go
func (c *Client) ListAccountTransactions(userGuid, accountGuid string) ([]*models.Transaction, error) {
   return ListAccountTransactionsWithDateRange(userGuid, accountGuid, "", "")
}
func (c *Client) ListAccountTransactionsWithDateRange(userGuid, accountGuid, fromDate, toDate string) ([]*models.Transaction, error) {
    ...
}
```